sympy/solvers/solveset.py

```diff
@@ -40,11 +40,11 @@ def invert_real(f_x, y, x):
     >>> from sympy import tan, Abs, exp
     >>> from sympy.abc import x, y, n
     >>> invert_real(exp(Abs(x)), y, x)
-    (x, {-log(y), log(y)})
+    (x, Intersection([0, oo), {log(y)}) U Intersection((-oo, 0], {-log(y)}))
```

Comment:
You should explain the idea behind this PR in the docstring, then maybe you should put these examples in a different "section".